### DISCUSSION TITLE
Problem CF swarm hl commander

### DISCUSSION THREAD

**Discussion by JuanYule** [bitcraze/discussions#130] [Q&A]
Hello,
I am trying to make a square trajectory (square) with the CFs, but I want to change the YAW at each corner of the square: I mean that at each corner the UAVs turn 90 degrees and follow a straight line. 

```
sequence_YAW = [
	[0.5, 1, 0.5, 0],
	[0.5, 1, 0.5, 90 * (np.pi/180)],
	[0.5, 3, 0.5, 90 * (np.pi/180)],
        [1.5, 3, 0.5, 90 * (np.pi/180)],
	[0.5, 3, 0.5, 0],
]
```
The drone executes the first command and the second one (turn 90 degrees), the problem is in the third command it does not work, and the drone stops working.

Does anyone know if the YAW can be changed in the hl commander?

**Comment by JuanYule** [comment]
I have already solved it

thx

**Reply by knmcguire** [reply]
Good to know! However, how did you solve it exactly? Just that we know what the initial issue was?
